doc/source/support/frequently-asked-questions.rst

```diff
@@ -134,6 +134,37 @@ Achieving the same result in Markdown often requires raw HTML, which is less rea
 
 Switching to ``README.rst`` at Level 5 helps users appreciate the formatting power of ``.rst`` and serves as a stepping stone toward writing full documentation in ``.rst`` as part of the ``scikit-package`` documentation standard.
 
+.. _faq-set-default-prompt-value:
+
+How can I change the default values that appear in the prompt when creating projects in level 3,4,5?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For example, to change some default values prompted when creating a new package in level 5, please follow the quick steps below:
+
+1. create ``~/.skpkgrc`` by runing ``touch ~/.skpkgrc``.
+2. copy and paste the following snippets.
+
+.. code-block:: json
+
+  {
+      "default_context":
+          {
+	       "maintainer_name": "Sangjoon Lee",
```

Comment:
Let's use all the examples I used in the PR. Again, that's the REAL user behavior. And remove `"project_needs_c_code_compiled": ["No", "Yes"],`.